frontend/src/app/main/ui/workspace/sidebar/assets.cljs

```diff
@@ -744,13 +788,25 @@
            (st/emit! (dwl/update-typography (merge typography changes) file-id))))
 
         handle-typography-selection
-        (fn [typography]
-          (let [attrs (merge
-                       {:typography-ref-file file-id
-                        :typography-ref-id (:id typography)}
-                       (d/without-keys typography [:id :name]))]
+        (fn [typography event]
+          (dom/stop-propagation event)
+
+          (swap! state update :selected
+                 (fn [selected]
+                   (cond
+                     (kbd/ctrl? event)
+                     (toggle-select selected (:id typography))
+
+                     (kbd/shift? event)
+                     (extend-select selected (:id typography) {"" typographies}))))
+
+          (let [ids (get-in @st/state [:workspace-local :selected])
```

Comment:
why access the state directly and not create a ref?

Reply:
Don't know, this also was already so before this branch :)